pinterest/teletraan
# Updating clone stage method arguments

Summary:
The ordering of the args in the clone_from_stage_name method was preventing stages from being cloned. This PR is to update that order.

Test Plan:

1. Navigate to a stage configuration

2. Create a stage by cloning
<img width="944" alt="image" src="https://github.com/pinterest/teletraan/assets/69278532/db6ebe95-c9d8-477c-95c8-796c18e9e7b9">


3. Confirm the stage was created with the selected stage type
<img width="1844" alt="image" src="https://github.com/pinterest/teletraan/assets/69278532/5405c9b4-1fec-45c7-b0dc-6202d119d70c">
